cfgov/unprocessed/apps/form-explainer/css/form-explainer.less

```diff
@@ -2,6 +2,28 @@
    "Form explainer" custom styles
    ========================================================================== */
 
+@gold-dark: #FF9E1B; // gold
```

Comment:
It seems not great to have a `@gold-dark` variable [in the DS](https://github.com/cfpb/design-system/blob/main/packages/cfpb-core/src/brand-colors.less#L123) that is `#dc731c`, and a `@gold-dark` variable here that is `#ff9e1b`. Despite the code comment, it is vulnerable to find and replace errors.

Could these accessible color adjustments be migrated to the DS?